web-share/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
```

Comment:
In this case, the implementor doesn't need to read the test unless they disagree with it. They just need to visit [the page](https://w3c-test.org/submissions/6287/web-share/share-url-empty-manual.html) and do what it says. The true/false here just determines whether the page prints out "Cancel the dialog" or "Choose a target".

So the call to `setupManualShareTest` should be treated with the same degree of clarity as any ordinary function call in a piece of software, not with some special level of clarity because implementors need to be able to read it. I would suggest we don't need every parameter to be tagged in every function call in JavaScript, and this is no exception. (But I'm happy to go with the flow here if there is consensus.)